gatewayconfig/gatewayconfig_app.py

```diff
@@ -96,10 +100,12 @@ def init_gpio(self, variant):
         on the detected hardware.
         """
         if is_raspberry_pi():
-            self.user_button = Button(self.get_button_gpio(), hold_time=USER_BUTTON_HOLD_SECONDS)
+            self.user_button = Button(
```

Comment:
And this linting is just wrong. We have a [120char limit](https://github.com/NebraLtd/hm-config/blob/bee8b9146f109cf6d45a76de3e3ce86d365f16c0/.github/linters/.flake8#L2).